test/test_transforms.py

```diff
@@ -121,3 +121,31 @@ def test_labeltoindex_jit(self):
         actual = transform_jit("test")
         expected = 0
         self.assertEqual(actual, expected)
+
+    def test_truncate(self):
```

Comment:
It is very important for unit tests to have doctoring so that future maintainers can get the idea of what this test is **supposed to** do.

Reply:
Done!